packages/design-system-react-native/src/components/Button/Button.types.ts

```diff
@@ -0,0 +1,29 @@
+import { ButtonBaseSize } from '../../primitives/ButtonBase';
+import type { ButtonPrimaryProps } from './variants/ButtonPrimary';
+import type { ButtonSecondaryProps } from './variants/ButtonSecondary';
+import type { ButtonTertiaryProps } from './variants/ButtonTertiary';
+
+export const ButtonSize = ButtonBaseSize;
```

Comment:
Oh nice! A new pattern for type aliasing has entered the chat 🤔 This might be the solution to the circular dependency struggles I’ve been facing!